package.json

```diff
@@ -7,8 +7,8 @@
     "url": "https://github.com/optro-cloud/optro-api-client.git"
   },
   "website": "https://www.optro.cloud",
-  "version": "1.0.5",
-  "license": "Commercial",
+  "version": "1.0.6",
+  "license": "MIT",
```

Comment:
I believe all our products should be "unlicensed" but we can check that another time.